ui/plugins/controller.js

```diff
@@ -329,9 +329,7 @@ treeherder.controller('PluginCtrl', [
             thBuildApi.cancelAllJobs($scope.repoName, rs.revision);
         };
 
-        /**
-         * Test whether or not the selected job is a reftest
-         */
+        // Test to expose the reftest button in the job details navbar
         $scope.isReftest = function() {
```

Comment:
@camd so I've tried it both ways. A new provider seems to work fine for logviewer, it also works for the job details navbar here above. However for the job details above, the reftest icon flashes off/on as the evaluation happens through the provider. I could be doing it incorrectly, but thinking maybe we leave this `$scope.isReftest` check as-is for optimal UX, and use the provider for logviewer. I'll push both so you can have a look and play with it locally.